src/Features/CSharp/Portable/GenerateMember/GenerateParameterizedMember/CSharpGenerateConversionService.cs

```diff
@@ -148,8 +148,7 @@ protected override bool IsValidSymbol(ISymbol symbol, SemanticModel semanticMode
         {
             methodSymbol = null;
             typeToGenerateIn = document.SemanticModel.GetTypeInfo(castExpression.Type, cancellationToken).Type as INamedTypeSymbol;
-            var parameterSymbol = document.SemanticModel.GetTypeInfo(castExpression.Expression, cancellationToken).Type as INamedTypeSymbol;
-            if (typeToGenerateIn == null || parameterSymbol == null || typeToGenerateIn.IsErrorType() || parameterSymbol.IsErrorType())
+            if (typeToGenerateIn == null || !(document.SemanticModel.GetTypeInfo(castExpression.Expression, cancellationToken).Type is INamedTypeSymbol parameterSymbol) || typeToGenerateIn.IsErrorType() || parameterSymbol.IsErrorType())
```

Comment:
feels like this is long enough to get some line breaks.

Reply:
sure